unstructured/documents/elements.py

```diff
@@ -146,6 +146,9 @@ class ElementMetadata:
     # MSFT Word specific metadata fields
     header_footer_type: Optional[str] = None
 
+    # Webpage & MSFT Word specific metadata fields
```

Comment:
Small nit, let's change this to "Formatting metadata fields". We don't do it now but we could infer that for PDFs/images too based on font information.

Reply:
Sounds good!